src/plugins/cal-events/cal-events.js

```diff
@@ -80,22 +79,31 @@ var componentName = "wb-calevt",
 	},
 
 	processEvents = function( $elm ) {
-		var settings = $.extend( {}, window[ componentName ], $elm.data( dataAttr ) ),
```

Comment:
You can't get rid of this line because it would remove the ability to extends the calendar